### PR TITLE
Table View MealDB query display on recipes tab

### PR DESCRIPTION
**Pull request by biennviquiera**
Implements table view cell information from mealDB API query method

### CODE REVIEW

**Comment by biennviquiera** [comment]
image
preview of the meal db table view

File: Disher/View Controllers/LoginViewController.m [hunk 1]
```diff
@@ -65,8 +65,8 @@ - (void) registerUser {
     newUser.password = self.passwordField.text;
     
     [newUser signUpInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {
-        if (error) {            
-            NSString *errorString = [error userInfo][@"error"];
+        if (error) {            NSString *errorString = [error userInfo][@"error"];
```
Comment (melissa-gu) [inline Disher/View Controllers/LoginViewController.m]: seems like an unintentional change we should undo

File: Disher/View Controllers/RecipesViewController.m [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
```
Comment (melissa-gu) [inline Disher/View Controllers/RecipesViewController.m]: ditto other comment

File: Disher/View Controllers/ListsViewController.m [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
```
Comment (melissa-gu) [inline Disher/View Controllers/ListsViewController.m]: Are we planning to override/implement these commented out methods in the future? If not we should get rid of them

File: Disher/View Controllers/RecipesViewController.m [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    NSURL *imageURL = [NSURL URLWithString:imageLink];
+    
+    [cell.recipeImage setImageWithURL:imageURL];
+    return cell;
+}
+
+- (IBAction)didTapLogout:(id)sender {
+    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
+        if (error) {
+
```
Comment (melissa-gu) [inline Disher/View Controllers/RecipesViewController.m]: let's add a TO DO here so we don't forget to handle the error

File: Disher/View Controllers/RecipesViewController.m [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+               [self presentViewController:alert animated: YES completion: nil];
+           }
+           else {
+               NSDictionary *dataDictionary = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
+               completionHandler(dataDictionary[@"meals"]);
+
+           }
+    }];
+    [task resume];
+
```
Comment (melissa-gu) [inline Disher/View Controllers/RecipesViewController.m]: super nit but let's start getting rid of unnecessary whitespace (here/elsewhere in the file)

**Review by melissa-gu — APPROVED** [review]
